spec/spec_helper.rb

```diff
@@ -14,3 +20,24 @@
     c.syntax = :expect
   end
 end
+
+class TestServiceProvider
```

Comment:
can thing go in its own file? I would never expect to find a class defined in `spec_helper` like this?

Reply:
Yup!